components/TextFields/src/MDCMultilineTextField.m

```diff
@@ -233,6 +236,25 @@ - (void)setupTextView {
   self.textView.opaque = NO;
 }
 
+// It's possible we've been passed a text view that's not our special class. We can handle that.
+// It's also something we guarantee to always be there. So, if it's nil, we make a new one.
+- (void)textViewSafetyCheck {
+  // We expect users of Interface Builder to put a UITextView in the view. When that happens,
+  // we swap it out for one of our special text views and bring the existing text over.
+  NSString *existingText = _textView.text;
```

Comment:
Do we care about any other state?

font?
textColor?
textAlignment?
etc.

Reply:
I was a little torn about those because of all the styling we apply. But I think you're right. The MDCMultilineTextField element is supposed to be as dumb as possible and that should include respecting whatever is set on its junk.

Reply:
Ugh. This isn't gonna work. I'm going to enforce they use my special class and be very verbose with instructions and asserts. This looks terrible!